### PR TITLE
lxde-base/menu-cache: CVE-2017-8933 and bump to 1.1.0

### PR DESCRIPTION
**Pull request by charIes17**

### CODE REVIEW

**Comment by charIes17** [comment]
@asarubbo 
As you opened the bug, could you please review?

**Comment by asarubbo** [comment]
LGTM

**Comment by Chiitoo** [comment]
There's also 1.0.0 out now: https://github.com/lxde/menu-cache/releases/tag/1.1.0

File: lxde-base/menu-cache/menu-cache-1.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Library to create and utilize caches to speed up freedesktop application menus"
+HOMEPAGE="http://lxde.sourceforge.net/"
+SRC_URI="mirror://sourceforge/lxde/${P}.tar.xz"
+
+LICENSE="GPL-2"
```
Comment (dracwyrm) [inline lxde-base/menu-cache/menu-cache-1.1.0.ebuild]: LICENSE="LGPL-2.1+" as per https://github.com/lxde/menu-cache/blob/master/COPYING They decided to switch it on us.

Comment (charIes17) [reply lxde-base/menu-cache/menu-cache-1.1.0.ebuild]: Done.

**Pull request closed without merging**